library/patient.inc.php

```diff
@@ -1563,7 +1563,7 @@ function getEffectiveInsurances($patient_id, $encdate)
         $insarr[] = $tmp;
     }
 
-    return $insarr;
+    return $insarr ?? array();
```

Comment:
This doesn't make sense to me.  How does $insarr ever get to be null?  Its defined as an array on Line 1551.  What's the error your seeing here in the logs?